Base/etc/fstab

```diff
@@ -10,5 +10,6 @@
 /www	/www	bind	bind,nodev,nosuid
 /usr/Tests	/usr/Tests	bind	bind,nodev,ro
 /usr/local	/usr/local	bind	bind,nodev,nosuid
+/usr/local/games	/usr/local/games	bind	bind,nodev,nosuid,wxallowed
```

Comment:
@awesomekling I'm a bit unsure about the security implications of this one...

Quake 3 makes heavy use of JIT compiled assembly as part of its "Virtual Machine" system: https://fabiensanglard.net/quake3/qvm.php
The game won't run without it. Obviously having this here presents somewhat of a nasty attack vector. Should we perhaps just tell the user to do this in `fstab` themselves?

Reply:
Yes, please add some instruction about this instead of making it the system default :)